### PR TITLE
Comic: Add a JSON description that can live forever

### PR DESCRIPTION
**Pull request by ekimekim**
The current images are generally too large to justify keeping around
forever by default. We solve this by splitting the generation of comics
into two stages. The first stage does all the random generation and spits
out an info object with a list of panels, character filenames, etc.
The second stage turns this info into an image.

We can convert this comic info to JSON and store it long-term.
As long as the source images are not deleted, we can then re-generate
the comic at any time. This lets us have a long-term archive of comics
(probably using the Lists module + some aliases) without wasted space.

As a bonus, this lets users give us arbitrary comic descriptions and we can render them.

### CODE REVIEW

**Review by Heufneutje — APPROVED** [review]
LGTM

**Comment by ekimekim** [comment]
I can't merge as I don't have write access, can someone merge pls?

**Pull request merged**
